test/blog.test.js

```diff
@@ -0,0 +1,83 @@
+var request = require("request"); // setting up the environment - requesting to the server
+var user = require("../app.js");
+var base_url = "http://localhost:3000/blog";
+
+xdescribe("Server",() => {
+  var server;
+  beforeALL(() => {
+    server = require("../app.js");
+    });
+    afterALL(() => {
+      server.close();
+  });
+});
+
+describe("myapp", function() {
```

Comment:
Would like to see all the these describes merged into one. DRY (Don't Repeat Yourself)